plugins/ci/pkg/ci/auto_detection.go

```diff
@@ -106,11 +106,6 @@ func ConfigFileAutoDetection(basePath string) (*models.Configuration, error) {
 			Paths: terraformPaths,
 		},
 	}
-	if len(terraformPaths) > 0 {
```

Comment:
@ivanfetch I might have missed the point of having it being enabled here... please, let me know the use-case